composer.json

```diff
@@ -68,6 +68,9 @@
             },
             {
                 "file": "app/config/samlstepupproviders_parameters.yml"
+            },
+            {
+                "file": "app/config/global_view_parameters.yml"
```

Comment:
How does incenteev handle 'deep parameters", ie. parameters that are objects (ie. `manual_url`)? IIRC one has to answer with JSON or something.